### DISCUSSION TITLE
Companion Mode

### DISCUSSION THREAD

**Discussion by Curly815** [NightscoutFoundation/xDrip#3127] [Q&A]
Hi everyone! I am using the Dexcom G7 on an S22 Ultra. I run Xdrip in companion mode because my endo requires Clarity from dexcom. I have noticed that it will stop reading for about 15min and then reconnects. Not sure why it does it. I think I have all the settings correct as I haven't had this issue before. I have a screenshot here of the event logs. Thanks for taking time to read and help!

![Screenshot_20231010_133934_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/71608093/de66d419-d932-480e-9d4b-454d081fb5df)

**Comment by Navid200** [comment]
Are you sure the Dexcom app continues to issue notifications when your phone screen is off or when the phone is locked?

**Reply by Curly815** [reply]
I think so. I pull down and it shows the reading even after xdrip disconnects and I have watched the numbers change also. All the dexcom notifications are turned also 🤷🏽‍♂️🤷🏽‍♂️

**Reply by Curly815** [reply]
![Screenshot_20231010_140354_Settings](https://github.com/NightscoutFoundation/xDrip/assets/71608093/19f8837e-e5b2-48d6-8333-71ef2ff0b69c)
![Screenshot_20231010_140404_Settings](https://github.com/NightscoutFoundation/xDrip/assets/71608093/0d21ddfe-59e0-4622-893a-437c228b528c)
